# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/RtcpEventObserver.h
# Commit: a1c16b86d3a7
# Full Hash: a1c16b86d3a7f328a690c00f86b837a8a67a656c
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Remove RtcpEventObserver. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102287
# ==============================================================================

diff -r c7dbdd82e8e8 -r a1c16b86d3a7 dom/media/webrtc/libwebrtcglue/RtcpEventObserver.h
--- a/dom/media/webrtc/libwebrtcglue/RtcpEventObserver.h	Mon Feb 01 09:28:23 2021 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,20 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef RTCP_EVENT_OBSERVER_
-#define RTCP_EVENT_OBSERVER_
-
-namespace mozilla {
-/**
- * This provides an interface to allow for receiving notifications
- * of rtcp bye packets and timeouts.
- */
-class RtcpEventObserver {
- public:
-  virtual void OnRtcpBye() = 0;
-  virtual void OnRtcpTimeout() = 0;
-};
-
-}  // namespace mozilla
-#endif